src/wp-admin/edit-form-blocks.php

```diff
@@ -65,8 +67,10 @@ static function ( $classes ) {
 	sprintf( '%s/autosaves?context=edit', $rest_path ),
 	'/wp/v2/settings',
 	array( '/wp/v2/settings', 'OPTIONS' ),
-	'/wp/v2/global-styles/themes/' . get_stylesheet(),
+	'/wp/v2/global-styles/themes/' . $stylesheet . '?context=view',
+	'/wp/v2/global-styles/themes/' . $stylesheet . '/variations?context=view',
 	'/wp/v2/themes?context=edit&status=active',
+	array( '/wp/v2/global-styles/' . WP_Theme_JSON_Resolver::get_user_global_styles_post_id(), 'OPTIONS' ),
 	'/wp/v2/global-styles/' . WP_Theme_JSON_Resolver::get_user_global_styles_post_id() . '?context=edit',
```

Comment:
Depending on user capabilities, a different [clientside request will fire](https://github.com/WordPress/gutenberg/blob/update%2Fglobal-styles-preload-paths-remove-6.6-compat/packages/editor/src/components/global-styles-provider/index.js#L87-L87), e.g., 

✅  user can update global styles: `'/wp/v2/global-styles/' . WP_Theme_JSON_Resolver::get_user_global_styles_post_id() . '?context=edit',`

🔕 user cannot update global styles: `'/wp/v2/global-styles/' . WP_Theme_JSON_Resolver::get_user_global_styles_post_id() . '?context=view',`

So at the moment, editors will get a clientside request since `'?context=view',` isn't preloaded.


I'm wondering if it's worth changing this to "view" context based on caps.

Probably for a follow up, as this already fixes a bunch of unnecessary requests.

Reply:
I've tried that here just to test things out:

- https://github.com/WordPress/gutenberg/pull/66541